src/app/home/landing-page/landing-page.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { LandingPageComponent } from './landing-page.component';
+
+describe('LandingPageComponent', () => {
+	let component: LandingPageComponent;
+	let fixture: ComponentFixture<LandingPageComponent>;
+
+	beforeEach(async(() => {
+		TestBed.configureTestingModule({
+			declarations: [ LandingPageComponent ]
```

Comment:
in some places we have `[items]` and in some `[ items ]`. which one is used more consistently?

Reply:
All the spec files are generated with `[ items ]` format, so I think that's fine